### ISSUE TITLE
output help if no arguments are given

### ISSUE THREAD

**Issue by dothebart** [GitTools/GitLink#58]
$ gitlink
GitLink v2.1.0.0
================

Arguments: 

Invalid number of arguments
An unexpected error occurred | [GitLinkException] GitLink.GitLinkException: Invalid number of arguments
   bei Catel.Logging.LogExtensions.ErrorAndThrowException[TException](ILog log, String messageFormat, Object[] args) in c:\CI_WS\Ws\5778\Source\Catel\src\Catel.Core\Catel.Core.Shared\Logging\Extensions\LogExtensions.cs:Zeile 398.
   bei GitLink.ArgumentParser.ParseArguments(List`1 commandLineArguments, IProviderManager providerManager)
   bei GitLink.Program.Main(String[] args)

which is pretty useless. ;-)

**Comment by GeertvanHorrik** [comment]
Agree. But it means you have a wrong command line. Can you post that one here?

**Comment by dothebart** [comment]
it was simply empty, as shown above. no arguments.

**Comment by GeertvanHorrik** [comment]
You must specify a directory:

```
GitLink.exe c:\source\catel
```

See docs:

https://github.com/GitTools/GitLink

**Comment by dothebart** [comment]
Hm, It seesm it doesn't work for me that way. We (arangodb) generate the vsd files using cmake in a fresh directory (i.e. Build64) which then for shure is not inside of git. Is there a way to specify the source root, and the build root? 
It also doesn't seem to like the projects generated: 
GitLink v2.1.0.0
================

Arguments: ./Build64/ -u https://github.com/arangodb/arangodb -b devel

Failed to load project 'C:\b\workspace\Contineous_W32_build\bbbb\ALL_BUILD.vcxproj': Das importierte Projekt "C:\Microsoft.Cpp.Default.props" wurde nicht gefunden. Vergewissern Sie sich, dass der Pfad in der <Import>-Deklaration korrekt und die Datei auf dem Datentraeger vorhanden ist.  C:\b\workspace\Contineous_W32_build\Build64\ALL_BUILD.vcxproj

I'm building it all that way: https://docs.arangodb.com/cookbook/CompilingUnderWindows.html

**Comment by GeertvanHorrik** [comment]
GitLink uses the VS project system to load the projects (in order to retrieve paths, etc). Apparently the imported props file cannot be found. Can you open this project / solution in VS?

**Comment by dothebart** [comment]
yes, it perfectly well opens in vs. Cmake / windows also uses msbuild to do the compiling. 
If you want to take a closer look:
https://github.com/arangodb/arangodb
Our windows build procedure:
https://docs.arangodb.com/cookbook/CompilingUnderWindows.html
